packages/block-editor/src/components/inserter/media-tab/media-tab.js

```diff
@@ -72,19 +72,11 @@ function MediaTab( {
 							render={ ( { open } ) => (
 								<Button
 									__next40pxDefaultSize
-									onClick={ ( event ) => {
-										// Safari doesn't emit a focus event on button elements when
-										// clicked and we need to manually focus the button here.
-										// The reason is that core's Media Library modal explicitly triggers a
-										// focus event and therefore a `blur` event is triggered on a different
-										// element, which doesn't contain the `data-unstable-ignore-focus-outside-for-relatedtarget`
-										// attribute making the Inserter dialog to close.
-										event.target.focus();
+									onClick={ () => {
 										open();
 									} }
 									className="block-editor-inserter__media-library-button"
 									variant="secondary"
-									data-unstable-ignore-focus-outside-for-relatedtarget=".media-modal"
```

Comment:
If this PR is merged, `data-unstable-ignore-focus-outside-for-relatedtarget` will not do anything, so we should remove it.